lib/client.js

```diff
@@ -378,8 +378,15 @@ class AssetComputeClient extends EventEmitter {
         }
         return response;
     }
-}
 
+    async checkEventJournal(journalUrl) {
+        const ioEvents = new AdobeIOEvents({
+            accessToken: this.accessToken,
+            orgId: this.integration.technicalAccount.org
+        });
+        await ioEvents.getEventsFromJournal(journalUrl);
```

Comment:
Based on the tests it seems the behavior is
1. if ready, no error
2. if not ready, throws an error

I think it would be nicer to return a boolean with true = ready and false = not ready.

And should we have a blocking and polling function (like `waitUntilJournalIsReady()`) or would the be covered elsewhere?

Reply:
In this use-case, `waitUntilJournalIsReady()` not needed as [in this PR](https://github.com/adobe/asset-compute-devtool/pull/22) the Run-click in UI is coded to make this `checkEventJournal()` call to validate if Journal is available.

Reply:
> as in success situation it also returns the events

This function does not return anything?

Reply:
aah right, currently `checkEventJournal` is not returning anything. 2 options:
1- change code to add return i.e 
```
const result =  await ioEvents.getEventsFromJournal(journalUrl); 
return result;
```
2- change API to return true/false

Though 1. might seem like a swissknife, but I think will move to true/false, as it does this job better.

Reply:
(2) is better, I think. You need to know if the journal is ready or not. No need to return events (which might also be a quite large object).

Reply:
true @tmathern !
this change might also help get rid of [the redundant code we discussed](https://github.com/adobe/asset-compute-devtool/pull/22#discussion_r455202017).

Reply:
That's where I was going ;)

Reply:
👍